cloud_functions/token_refresh/tests/test_main.py

```
@@ -0,0 +1,238 @@
+         patch("cloud_functions.token_refresh.utils.cloud_utils.get_secret", side_effect=get_secret_side_effect), \
+         patch.dict(os.environ, mock_env), \
+         patch("cloud_functions.token_refresh.utils.cloud_utils.secretmanager.SecretManagerServiceClient", return_value=mock_secret_manager), \
+         patch("cloud_functions.token_refresh.streams.strava.get_secret", side_effect=get_secret_side_effect), \
+         patch("cloud_functions.token_refresh.streams.strava.encrypt_token", side_effect=lambda token, _: f"encrypted_{token}"):
+        
+        # Set up the mock client instance
+        mock_client_instance = mock_client_class.return_value
+        mock_client_instance.refresh_access_token.return_value = mock_strava_response
+
+        # Verify it raises the correct error
+        with pytest.raises(ValueError) as exc_info:
```

Ruff (0.8.2)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)